src/box/lua/stat.c

```diff
@@ -67,6 +67,23 @@ inject_current_stat(struct lua_State *L, const char *name, size_t val)
 	lua_pop(L, 1);
 }
 
+/**
+ * Function creates table, which contains one field `total`
+ * with given @val. Puts this table to the field @name in
+ * the table which located on the top of the lua stack.
+ */
+static void
+inject_total_stat(struct lua_State *L, const char *name, size_t val)
+{
+	lua_pushstring(L, name);
+	lua_newtable(L);
+	lua_pushstring(L, "total");
+	lua_pushnumber(L, val);
+	lua_settable(L, -3);
+	lua_settable(L, -3);
```

Comment:
Please be consistent and use rawset everywhere.

Reply:
fixed